socorro/cron/crontabber_app.py

```diff
@@ -99,6 +98,25 @@ def jobs_converter(path_or_jobs):
 
 
 class CronTabberApp(CronTabberBase, App):
+    config_defaults = {
+        'always_ignore_mismatches': True,
+
+        'crontabber': {
+            'database_class': 'socorro.external.postgresql.connection_context.ConnectionContext',
+
+            'class-BugzillaCronApp': {
+                'days_into_past': 0,
+            },
```

Comment:
I think this is actually in the wrong namespace and has no effect on anything (this is true in consul for prod as well as the `crontabber.env` file).

Also, this is the default value:

https://github.com/mozilla-services/socorro/blob/d442a4df44a2be121a0ad35a7c3151afc885263a/socorro/cron/jobs/bugzilla.py#L95

We can nix the `class-BugzillaCronApp` clause.